ui/test/nuxeo-directory-widgets.test.js

```diff
@@ -172,63 +172,93 @@ suite('nuxeo-directory-suggestion', () => {
         ],
       },
     ];
-    suite('bound to directory data with computed ids', () => {
-      setup(async () => {
-        server.respondWith('POST', '/api/v1/automation/Directory.SuggestEntries', [
-          200,
-          { 'Content-Type': 'application/json' },
-          JSON.stringify(response),
-        ]);
-      });
+    setup(async () => {
+      server.respondWith('POST', '/api/v1/automation/Directory.SuggestEntries', [
+        200,
+        { 'Content-Type': 'application/json' },
+        JSON.stringify(response),
+      ]);
+    });
+
+    test('Then I can select an entry', async () => {
+      suggestionWidget = await fixture(html`
+        <nuxeo-directory-suggestion directory-name="l10nsubjects" multiple min-chars="0"></nuxeo-directory-suggestion>
+      `);
+      await assertSelectedValue(suggestionWidget, 'ar', 'Architecture');
+    });
 
-      test('Then I can select an entry', async () => {
-        suggestionWidget = await fixture(html`
-          <nuxeo-directory-suggestion directory-name="l10nsubjects" multiple min-chars="0"></nuxeo-directory-suggestion>
-        `);
-        await assertSelectedValue(suggestionWidget, 'ar', 'Architecture');
-      });
+    test('Then I can select and re-select an entry', async () => {
```

Comment:
This was actually the only new test I added. I just removed a suite that wasn't doing anything and the rest is indentation.